extensions/notebook/src/common/utils.ts

```diff
@@ -251,6 +259,15 @@ export async function exists(path: string): Promise<boolean> {
 	}
 }
 
+export function existsSync(path: string): boolean {
```

Comment:
Why do we need this when we have fs.pathExistsSync() ?

Reply:
Previously the exists method (line 252) was being called - not sure why fs.pathExists() wasn't used. But I can change it and remove this method.